src/shared/sentry/sentryadapter.cpp

```diff
@@ -4,7 +4,10 @@
 
 #include "sentryadapter.h"
 
-#include <QtQml/private/qqmlengine_p.h>
+#ifndef UNIT_TEST
+// This is not found when building tests ... wierd.
```

Comment:
https://github.com/mozilla-mobile/mozilla-vpn-client/pull/7176/commits/dfe8c8696fbfa2b322e5aced6b30d7a00c785a5d#diff-75169c8db799911a8b0af81ad1c17c24e41a868d224d756d249b7f8b070cee0c

tried that before, did not work :c `auth_test` did not build. In theory given sentry is part of shared-sources and that is an interface target, it "should" build. Also locally that builds too, so i blame something in github ci. 

>How come it's not necessary in tests?
Because i'm also "#ifndef UNIT_TEST" ing the code that calls this

Reply:
Got it. Hum, maybe because the code is not called ever it's being optimized away. Who know, looks correct to not import it in this case anyways.

Reply:
I guess this comment can go though.